### ISSUE TITLE
Perform validation for all popup-dialogues

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#284]
```
Just like issue #273 it is possible to do the same on several other dialogues, 
such as the level up (giving you negative experience) and buying items (you get 
more items than they sold).

For more information see issue issue #273.
```

Original issue reported on code.google.com by `Jimmy.Ax...@gmail.com` on 25 Nov 2011 at 11:37

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. This is now fixed with r222.
```

Original comment by `oskar.wi...@gmail.com` on 12 Mar 2012 at 9:18
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
